src/applications/personalization/search-representative/config/chapters/search/searchRepresentativeReviewWidget.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+const SearchRepresentativeReviewWidget = props => {
+  const capitalizeFirstLetter = string => {
+    return string.charAt(0).toUpperCase() + string.slice(1);
+  };
+
+  return (
+    <>
+      {Object.keys(props?.formData?.representativeData).map((item, index) => {
+        return (
+          <>
+            <div key={index} className="vads-u-display--flex">
+              <p className="vads-u-flex--1 vads-u-text-align--left vads-u-font-weight--normal vads-u-margin-y--0">
+                {capitalizeFirstLetter(item)}:
```

Comment:
I think we can remove the semicolon here

Reply:
Ugh, prettier somehow slipped this in at the last second. Removed.